tests/acceptance/core/testcase/pod.go

```diff
@@ -0,0 +1,49 @@
+package testcase
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/k3s-io/k3s/tests/acceptance/core/service/assert"
+	"github.com/k3s-io/k3s/tests/acceptance/shared/util"
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+)
+
+// TestPodStatus test the status of the pods in the cluster using 2 custom assert functions
+func TestPodStatus(
+	g ginkgo.GinkgoTInterface,
```

Comment:
This argument is unused and is overridden by `g gomega.Gomega` for most of the function.